content/docs/get-started/aws/begin.md

```diff
@@ -50,17 +52,37 @@ Next, you'll install the required language runtime, if you haven't already.
 {{< install-dotnet >}}
 {{% /choosable %}}
 
-Finally, you'll configure AWS.
+Finally, you'll configure Pulumi with AWS.
 
-### Configure AWS
+### Configure Pulumi to access your AWS account
 
-<a href="{{< relref "/docs/intro/cloud-providers/aws/setup" >}}" target="_blank">Configure AWS</a> so the Pulumi CLI can connect to AWS. If you have previously configured the <a href="https://aws.amazon.com/cli/" target="_blank">AWS CLI</a>, `aws`, Pulumi will respect and use your configuration settings.
+Pulumi requires cloud credentials to manage and provision resources. You must use an IAM user account that has **Programmatic access** with rights to deploy and manage resources handled through Pulumi. In this guide, your IAM user must have the `s3:CreateBucket` IAM policy action granted.
 
-If you have multiple AWS profiles set up, specify a different profile using one of the following ways:
+If you have previously <a href="https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html" target="_blank">installed</a> and <a href="https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html" target="_blank">configured</a> the AWS CLI, Pulumi will respect and use your configuration settings.
+
+If you do not have the AWS CLI installed or plan on using Pulumi from within a CI/CD pipeline, <a href="https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys" target="_blank">retrieve your access key ID and secret access key</a> and then set the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables on your workstation.
+
+#### Linux/macOS
```

Comment:
We have a handful of component-backed shortcodes that handle toggling this kind of content automatically. Unless there's a need to show all variations to the same user (which should be rare), you should opt for the Hugo shortcodes `{{% chooser %}}` and `{{% choosable %}}` instead. [Docs here](https://github.com/pulumi/docs/tree/master/components#available-components).

Reply:
👍 I pulled these from https://www.pulumi.com/docs/intro/cloud-providers/aws/setup/#environment-variables, so I'll update both locations with the change.